rules_conditions.md

```diff
@@ -97,6 +97,23 @@ Conditions:
     Datatype: int
 ```
 
+## Adding Span Context to Conditions
+
+You can limit conditions to a particular span context. Right now, the `root` prefix is the only context supported. 
+This limits matching spans to those where the root span matches the condition. Note that this behavior obsoletes matching 
+specific spans within a trace since the entire trace shares a root span. 
```

Comment:
```suggestion
Fields can contain a span selection prefix. Today, the only prefix supported is `root`.
This prefix causes the root span to be searched for the specified field, rather than the span being evaluated.
```